src/Microsoft.DotNet.Interactive.PowerShell/PowerShellExtensions.cs

```diff
@@ -96,5 +97,32 @@ internal static SecureString GetSecureStringPassword(this PasswordString pwdStri
 
             return secure;
         }
+
+        internal static object Unwrap(this PSObject inputObject)
+        {
+            object obj;
+            if (inputObject is PSObject psObj)
```

Comment:
`inputObject` is already a `PSObject`.
Please refactor this method to something like the following:
```c#
public static object Unwrap(this PSObject psobj)
{
    object obj = psObj.BaseObject;
    if (obj is PSCustomObject)
    {
        ....
        obj = table;
    }

    return obj;
}
```